impact_deploy/recipes/install_keys.rb

```diff
@@ -0,0 +1,43 @@
+require 'chef/log'
+Chef::Log.level = :debug
+impact_environment = node['deploy']['mc']['environment']['IMPACT_ENVIRONMENT']
+ecs_secret_access_key = node['deploy']['mc']['environment']['ECS_SECRET_ACCESS_KEY']
+ecs_access_key_id = node['deploy']['mc']['environment']['ECS_ACCESS_KEY_ID']
+ecs_default_region = node['deploy']['mc']['environment']['AWS_DEFAULT_REGION']
+script "install_keys" do
+  interpreter "bash"
+  user "deploy"
+  cwd "/home/deploy"
+  environment node['deploy']['mc']['environment']
+  code <<-EOH
+    # trigger a deploy of impact-api
+    export $DEPLOY_KEY
+    export IMPACT_ENVIRONMENT=#{impact_environment}
+    export AWS_DEFAULT_REGION=#{ecs_default_region}
+    export ECS_SECRET_ACCESS_KEY=#{ecs_secret_access_key}
+    export ECS_ACCESS_KEY_ID=#{ecs_access_key_id}
+    export AWS_ACCESS_KEY_ID=#{ecs_access_key_id}
+    export AWS_SECRET_ACCESS_KEY=#{ecs_secret_access_key}
+    if [ -z "$IMPACT_ENVIRONMENT" ]; then 
+    	echo "IMPACT_ENVIRONMENT not set. install keys skipped";
+    else
+        virtualenv --no-site-packages .venv && source .venv/bin/activate
+        pip install awscli --upgrade
+        .venv/local/bin/aws s3api get-object --bucket masschallenge-deployment --key secure/ecs-key.pem ~/.ssh/ecs-key.pem
+        export ECS_PEM_FILE=~/.ssh/ecs-key.pem
+        export ECS_INSTANCES=`.venv/local/bin/aws ecs list-container-instances --cluster $IMPACT_ENVIRONMENT | grep arn | cut -b 64-99 | xargs`
```

Comment:
It would've been clearer to me if the variable name would clarify that the instances are of containers, not of the hardware ec2.

Reply:
renamed